src/sbvr-api/sbvr-utils.ts

```diff
@@ -57,6 +59,7 @@ import {
 	statusCodeToError,
 	TranslationError,
 	UnauthorizedError,
+	AnyObject,
```

Comment:
Doesn't need to happen with with PR, but feels like `AnyObject` should be in `common_types.ts`, definitely not in `errors.ts`.